repos.d/kiss.yaml

```diff
@@ -37,18 +37,18 @@
     - name: repo
       fetcher: GitFetcher
       parser: KissGitParser
-      url: 'https://github.com/kisslinux/community.git'
+      url: 'https://github.com/kiss-community/repo-community.git'
```

Comment:
This needs git branch specified as it's no longer assumed `master`.
Also the parsing fails on 
https://github.com/kiss-community/repo-community/blob/main/community/tcc/version